Frontend/ui-driver/index.js

```diff
@@ -2,6 +2,8 @@ import "core-js";
 import "presto-ui";
```

Comment:
can we please include bundle load time timestamp which is very much required, you can add a timeStamp in main becknbase.html which will be the timeStamp from `native initiate to bundle` time and in first line of `index.js` which will be the time for `bundle load`
flow will be like `native initiate -> becknbase.html -> index.js`

Reply:
done